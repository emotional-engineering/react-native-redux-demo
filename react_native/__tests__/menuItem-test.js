import React from 'react';
import MenuItem from '../elements/menuElement';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<MenuItem onPress={()=>{}} />).toJSON();
  expect(tree).toMatchSnapshot();
});
