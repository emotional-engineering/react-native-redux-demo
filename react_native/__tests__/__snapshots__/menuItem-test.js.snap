// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  onPress={[Function]}
  style={
    Object {
      "backgroundColor": "#6563A3",
      "borderRadius": 10,
      "color": "#ffffff",
      "fontSize": 28,
      "height": 60,
      "lineHeight": 60,
      "marginLeft": 10,
      "marginRight": 10,
      "marginTop": 10,
      "textAlign": "center",
    }
  }
/>
`;
